no-computer.md

```diff
@@ -58,12 +123,37 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since Diplomacy already has a syntax for submitting instructions, it would be
+best to use the same syntax.
+However, we would need to create an intuitive interface to store the map.
+A map can be easily translated into a graph and 
+there are a lot of ways of representing graphs.
+So I think it would not be too difficult to choose a representation and build
+an interface for entering the map's data.
+
```

Comment:
Would representing the graphs be a language design aspect? Or is this project more focused on using the pre-existing language.